### DISCUSSION TITLE
DevOpsTools file showing error and make mint ARGS="--network sepolia" not working due to no contract returned

### DISCUSSION THREAD

**Discussion by Aishwaria8786** [Cyfrin/foundry-full-course-cu#293] [Q&A] [answered]
![Screenshot (266)](https://github.com/Cyfrin/foundry-full-course-f23/assets/68544977/c0c9e5b1-70b6-4ff2-9fdc-a2e06c238030)
What to do to solve this issue?

**Comment by Niferu** [comment]
Hello @Aishwaria8786 

Update your mapping in `foundry.toml` to below:
```toml
remappings = ["foundry-devops/=lib/foundry-devops"]
```

**Comment by shikhar229169** [comment]
I think your foundry-devops folder is not in ```lib``` folder.
So just move the ```foundry-devops``` folder to ```lib``` folder.

Refer to this image:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/111875681/6ccf9e1a-f35b-40a5-9af3-8fb62d9bffb4)

**Comment by ChefAharoni** [comment]
Remappings can be tricky, foundry is a little bit buggy in VSCode.

Try these [solutions](https://github.com/Cyfrin/foundry-full-course-f23/discussions/194) regarding mappings issues.

For me, writing the remappings in a .txt file helped, this is how it looks in mine:
<img width="378" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/4399057/dda8f167-0918-4206-8ea1-538e980e49ed">

```text
@openzeppelin/=lib/openzeppelin-contracts/
```

You can also run `forge remappings` to see the remappings foundry is recognizing.

**Comment by cromewar** [comment]
Also please refer to #223 for more context on this error

**Comment by Aishwaria8786** [comment, marked as answer]
Thank you all.I found solution this way #169 .
